### ISSUE TITLE
'Related Resources' CKAN item integration on Briefing pages

### ISSUE THREAD

**Issue by DBishton** [oeco/opendev-wptheme#22]
This feature is working well, but requires a little tweaking in terms of display and functionality. Currently the 'Read More...' link takes users to the top of the page they are currently on, which makes it appear to do nothing. Please clarify the intended operation, and perhaps remove this link in the interim.

This is also the current behaviour when a user clicks on the title of a CKAN dataset. E.g. 'The Lower Mekong Dry Forest Ecoregion'. Suggest this takes the user directly to that dataset on CKAN rather than the current page. 

![image](https://cloud.githubusercontent.com/assets/1206387/6125218/69e5fbd6-b146-11e4-8534-f550e5e4b37b.png)

Items in more than one CKAN group are also appearing twice in this group.

See example at: http://opendevelopmentmekong.net/briefings/forests-and-forestry/

**Comment by miguelpeixe** [comment]
Hi @DBishton 

This duplication of datasets is a bigger design issue. I thought the groups were more of a major categorization, that datasets wouldn't be part of multiple groups. Not being the case, we might need to think other visualization for this section. This design only makes sense if we see a clear separation between the datasets, something that doesn't intersect. Not only the duplication of content, but "Cambodia" and "Library" being in the same taxonomy group of this section is a bit strange looking.

**Comment by DBishton** [comment]
Hi @miguelpeixe, we do need to resolve this duplication issue, and present these resources in a way that does not take up so much screen real estate. @acorbi -- do you have a suggestion on the duplication issue? My suggestion for the screen space would be a more 'collapsed' view as default?

See the screenshot below when many items are associated with a briefing page -- some content of the page needs to be visible, regardless of the number of related resources.

![image](https://cloud.githubusercontent.com/assets/1206387/6391605/94b84d2c-bdea-11e4-83af-708377703dea.png)

**Comment by WillieShubert** [comment]
As Miguel noted, the duplication of content in CKAN is the bigger issue causing the duplication since the same content can be found across multiple folders. This is not a design issue nor is it a JEO issue but rather a content structure issue that must be addressed by the persons responsible for CKAN content management. 

Trying to solve this issue with a design change will result in the reduction in the quality of two systems  (Wordpress and CKAN) in order to cover over the problem about how content is organized in CKAN.

**Comment by acorbi** [comment]
@DBishton yeah, this is a tricky one. As i see it (and @WillieShubert stated), it would be complex to solve this programatically. On the other hand i do not see clearly how contents could be re-structured in CKAN in order to remove this duplicity ( Currently it is quite consistent that datasets are assigned to a thematic group, such as Laws, and to the country group they belong to, in this example Cambodia)

@miguelpeixe Any ideas for implementing a filter that removes duplicates while retrieving datasets from CKAN per plugin's shortcodes?

**Comment by acorbi** [comment]
During discussion with @WillieShubert, we came up with a possible solution which might involve using the CKAN dataset's extra fields to store the location the information is related to in order to filter out the contents by this field and not by group, as it is currently happening. This will involve updating the wpckan with the feature described in https://github.com/OpenDevelopmentMekong/wpckan/issues/73.

The current workflow offers editors a **spatial range** field (http://wiki.opendevelopmentmekong.net/metadata#open_development_network_metadata_template) while creating/editing datasets on CKAN and content in Wordpress is being already geotagged thanks to JEO's provided functionalities.
